kotlin/configure-subnetwork-trace/README.md

```diff
@@ -25,7 +25,7 @@ Example barrier conditions for the default dataset:
 5. Update the selected barrier expression and the checked options in the UI using this tier's `TraceConfiguration`.
```

Comment:
oops sorry I should have spotted this before, this should be `UtilityTraceConfiguration` (all of our UtilityNetwork API classes have the prefix of `Utility`, just for future note!)

Reply:
sorry to add more changes on but since we're editing this readme anyway, I also noticed on line 23 there is a reference to `Definition.networkAttributes`. Could you change this to `UtilityNetworkDefinition` please?

Reply:
Thanks, will keep checking for these!